statsmodels/regression/linear_model.py

```diff
@@ -2637,9 +2644,10 @@ def get_robustcov_results(self, cov_type='HC1', use_t=None, **kwargs):
                 raise ValueError('either time or groups needs to be given')
             groupidx = lzip([0] + tt, tt + [nobs_])
             self.n_groups = n_groups = len(groupidx)
-            res.cov_params_default = sw.cov_nw_panel(self, maxlags, groupidx,
-                                                     weights_func=weights_func,
-                                                     use_correction=use_correction)
+            res.cov_params_default = \
+                sw.cov_nw_panel(self, maxlags, groupidx,
+                                weights_func=weights_func,
+                                use_correction=use_correction)
```

Comment:
that's the only formatting change that is annoying, especially because it's not any standard formatting either.